### PR TITLE
Query and update animals table pull table

### PR DESCRIPTION
**Pull request by alicemirigo92**
- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

image
image
image
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -17,4 +17,54 @@ SELECT * FROM animals WHERE neutered = true;
...
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon and setting species to pokemon for all animals that don't have a species already set inside a transaction:
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```
Comment (Ghiftee) [inline queries.sql:35]: - [x] Before committing, kindly verify that the change was made as you did above. e.g 

``` 
SELECT species from animals; 

COMMIT;

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @alicemirigo92 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ Screenshots are added
✔️ Queries are successful
✔️ Gitflow is used

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql (hunk 1 above)
Comment (alicemirigo92) [reply queries.sql:35]: @Ghiftee, thank you for the review, i have added a screenshot for that.

Comment (danmainah) [reply queries.sql:35]: - [x] @alicemirigo92 you are supposed to add the code in the queries as well as showcase it in the screenshot, kindly add the verification that the change was done before the commit as previously requested

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @alicemirigo92 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Queries are successful
✔️ Github flow was correctly used

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @alicemirigo92, 

To highlight 👍🏽 
✔️ All queries are added
✔️ No errors

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**